esx_service/vsan_policy.py

```diff
@@ -87,6 +89,7 @@ def delete(name):
     try:
         os.remove(policy_path(name))
     except:
+        logging.exception("Failed to remove %s policy file", name)
```

Comment:
nit: not sure this one is needed , if the file does not exist silent fail is better. But it's good as is as well